pkg/cmd/cli/cmd/rollback.go

```diff
@@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	}
 
 	cmd := &cobra.Command{
-		Use:     "rollback DEPLOYMENT",
+		Use:     "rollback [dc/]config-name",
 		Short:   "Revert part of an application back to a previous deployment",
 		Long:    rollbackLong,
 		Example: fmt.Sprintf(rollbackExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
 			// Validate arguments
 			if len(args) == 0 || len(args[0]) == 0 {
-				cmdutil.CheckErr(cmdutil.UsageError(cmd, "A deployment name is required."))
+				cmdutil.CheckErr(cmdutil.UsageError(cmd, "A deploymentConfig name is required."))
```

Comment:
"Deployment config" or "Deployment configuration"

No human readable message should ever contain a camel case name except in
extremely specific circumstances (truly generic messages.

On Jul 9, 2015, at 10:59 AM, David Eads <notifications@github.com> wrote:

In pkg/cmd/cli/cmd/rollback.go
<https://github.com/openshift/origin/pull/3643#discussion_r34264323>:

>  		Short:   "Revert part of an application back to a previous deployment",
>  		Long:    rollbackLong,
>  		Example: fmt.Sprintf(rollbackExample, fullName),
>  		Run: func(cmd *cobra.Command, args []string) {
>  			// Validate arguments
>  			if len(args) == 0 || len(args[0]) == 0 {
> -				cmdutil.CheckErr(cmdutil.UsageError(cmd, "A deployment name is required."))
> +				cmdutil.CheckErr(cmdutil.UsageError(cmd, "A deploymentConfig name is required."))

DeploymentConfig, deploymentconfig, or dc please.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3643/files#r34264323>.